android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -65,6 +68,48 @@ public TiListView(ListViewProxy proxy)
 		this.recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
 		this.recyclerView.setFocusableInTouchMode(false);
 
+		// Add listener to fire scroll events.
+		this.recyclerView.addOnScrollListener(new RecyclerView.OnScrollListener()
+		{
+			@Override
+			public void onScrollStateChanged(@NonNull RecyclerView recyclerView, int newState)
+			{
+				super.onScrollStateChanged(recyclerView, newState);
+
+				if (newState == RecyclerView.SCROLL_STATE_IDLE) {
+					isScrolling = false;
+					proxy.fireSyncEvent(TiC.EVENT_SCROLLEND, generateScrollPayload());
+				}
```

Comment:
If you set `ListView.canScroll` to `false` and slide your finger on the `ListView`, it'll fire a "scrollend" event when it shouldn't. We probably should do the below...
```javascript
if (isScrolling && (newState == RecyclerView.SCROLL_STATE_IDLE)) {
	isScrolling = false;
	proxy.fireSyncEvent(TiC.EVENT_SCROLLEND, generateScrollPayload());
}
```